src/custom/state/orders/updater.ts

```diff
@@ -150,47 +151,45 @@ export function EventUpdater(): null {
         topics: eventTopics
       })
 
-      // only update lastCheckedBlock if no logs
-      if (logs.length === 0) {
-        // SET lastCheckedBlock = lastBlockNumber
-        dispatch(updateLastCheckedBlock({ chainId, lastCheckedBlock: lastBlockNumber }))
-        return
-      }
-
       const block2DateMap = await buildBlock2DateMap(library, logs)
 
+      const ordersBatchData: (OrderFulfillmentData & Pick<Log, 'transactionHash'>)[] = logs.map(log => {
```

Comment:
i would avoid complex inline ts expressions
the line has already a declaration and a map, adding this makes it harder to read